opm/output/eclipse/summaryComparator.cpp

```diff
@@ -0,0 +1,243 @@
+/*
+  Copyright 2016 Statoil ASA.
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+
+#include <opm/output/eclipse/summaryComparator.hpp>
+#include <ert/ecl/ecl_sum.h>
+#include <ert/util/stringlist.h>
+#include <ert/util/int_vector.h>
+#include <ert/util/bool_vector.h>
+#include <opm/common/ErrorMacros.hpp>
+
+bool SummaryComparator::open(const char* basename1, const char* basename2){
```

Comment:
I don't fully understand what you are trying to say here. In the current constructor. there is a call for `init(const, char*, const char*)`  which calls this open(..)-function. Do you want the functionallity of the open-function implemented directly in the constructor (and no open-function)?